pages/dkp/kommander/2.2/install/air-gapped/catalog/index.md

```diff
@@ -1,110 +1,255 @@
+           address-pools:
+             - name: my-ip-space
+               protocol: bgp
+               addresses:
+                 - 172.40.100.0/24
+   ```
+
+In the above configuration, `peers` defines the configuration of the BGP peer, such as peer IP address and `autonomous system number` (`asn`).
+The `address-pools` section is similar to `layer2`, except for the protocol.
+
+MetalLB also supports [advanced BGP configuration][metallb_config].
+
+See [Kommander Load Balancing][kommander-load-balancing] for more information.
+
+### Determine the installation version
+
+Set the `VERSION` environment variable to the version of Kommander you want to install, for example:
+
+```bash
+export VERSION=v2.2.0
+```
+
+### Load the Docker images into your Docker registry
+
+1.  Download the Kommander image bundle file:
 
     ```bash
-    export TARGET_NAMESPACE=<WORKSPACE_OR_PROJECT_NAMESPACE>
+    wget "https://downloads.mesosphere.com/kommander/airgapped/${VERSION}/kommander_image_bundle_${VERSION}_linux_amd64.tar.gz" -O kommander-image-bundle.tar.gz
     ```
 
-1.  Go to the Gitea UI below and register a new user account:
+1.  Download the [DKP catalog applications][dkp_catalog_applications] image bundle file:
 
     ```bash
-    GITEA_HOSTNAME=$((kubectl -n kommander get cm konvoyconfig-kubeaddons -o go-template='{{if ne .data.clusterHostname ""}}{{.data.clusterHostname}}{{"\n"}}{{end}}' ; kubectl -n kommander get ingress gitea -o jsonpath="{.status.loadBalancer.ingress[0]['ip','hostname']}") | head -1) && echo https://${GITEA_HOSTNAME}/dkp/kommander/git/
+    wget "https://downloads.mesosphere.com/kommander/airgapped/${VERSION}/catalog_applications_image_bundle_${VERSION}_linux_amd64.tar.gz" -O catalog-applications-image-bundle.tar.gz
     ```
 
-1.  Create a new repository under the new user account. Rest of this guide assumes you created a private repository named `dkp-catalog-applications` with `main` as default branch. You can substitute these values as needed. Create environment variables that contain the Gitea credentials and repository metadata:
+1.  Place the bundles in a location where you can load and push the images to your private Docker registry.
+
+1.  Run the following command to load the air-gapped image bundle into your private Docker registry:
 
     ```bash
-    GITEA_USERNAME=<YOUR_GITEA_USERNAME>
-    GITEA_PASSWORD=<YOUR_GITEA_PASSWORD>
-    GITEA_REPOSITORY_NAME=dkp-catalog-applications
-    GITEA_REPOSITORY_DEFAULT_BRANCH=main
+    dkp push image-bundle --image-bundle kommander-image-bundle.tar.gz --to-registry <REGISTRY_URL>
+    dkp push image-bundle --image-bundle catalog-applications-image-bundle.tar.gz --to-registry <REGISTRY_URL>
     ```
 
-1.  Clone the newly created repository on your local machine:
+It may take a while to push all the images to your image registry, depending on the performance of the network between the machine you are running the script on and the docker registry.
```

Comment:
```suggestion
It may take a while to push all the images to your image registry, depending on the performance of the network between the machine you are running the script on and the Docker registry.
```